src/store/modules/util/getters.ts

```diff
@@ -2,5 +2,9 @@ import { GetterTree } from 'vuex'
 import UtilState from './UtilState'
 import RootState from '@/store/RootState'
 
-const getters: GetterTree <UtilState, RootState> = {}
+const getters: GetterTree <UtilState, RootState> = {
+  getServiceStatusDesc(state) {
```

Comment:
Update this getter to accept the status id and it will return the desc.

Reply:
Updated getter Sir.